key_action_map.h

```diff
@@ -21,12 +21,18 @@ class key_action_map
   ///Find out which action is triggered by that key
   action_type to_action(sf::Keyboard::Key key) const noexcept;
 
+  /// Find out which key triggers that action
+  sf::Keyboard::Key to_key(action_type action) const noexcept;
+
   ///Find out if the key is mapped
   bool has_key(sf::Keyboard::Key key) const noexcept;
 
   ///Returns const ref of m_map
   const std::map<sf::Keyboard::Key, action_type>& get_raw_map() const noexcept {return m_map;}
 
+  // Find the stun key
```

Comment:
This should one day be a free function, as we can already get this info from the `get_raw_map` member function result. I think I saw an Issue about that even already :-)

Reply:
Ah yes good catch, this is a leftover of me addressing #304, then reverting my progress to let someone more junior resolve it. I forgot to remove this bit.
Now that we have `key_action_map::to_key` (see #355), solving #304 should be trivial. Should I make it a `first` issue?

Reply:
Making that a `first` Issue is great! A `first` Issue always has the tests written, as that is what makes it a `first`. I predict you'll happily write these as well :-)

Reply:
I put the test back 👍